packages/optimizely-sdk/lib/core/event_processor/index.ts

```diff
@@ -22,6 +22,6 @@ export function createEventProcessor(
   return new LogTierV1EventProcessor(...args);
 }
 
-export { EventProcessor, LocalStoragePendingEventsDispatcher } from '@optimizely/js-sdk-event-processor';
+export type { EventProcessor, LocalStoragePendingEventsDispatcher } from '@optimizely/js-sdk-event-processor';
```

Comment:
Suggest removing `LocalStoragePendingEventsDispatcher` - as we discussed, nothing is using it, and I got confused thinking this change would cause a problem.
```suggestion
export type { EventProcessor } from '@optimizely/js-sdk-event-processor';
```